### DISCUSSION TITLE
Transition to Vite after create-react-app deprication?

### DISCUSSION THREAD

**Discussion by hadi-khir** [UI5/webcomponents-react#4708] [Ideas]
Hey team!

I really love the seed template you guys have created, and I was wondering, now that create-react-app is no longer recommended by the react-core team, if there will be any transition to a new template? 

Best,
Hadi

**Comment by MarcusNotheis** [comment]
Hey @hadi-khir,

actually we already have a vite template using TypeScript ready: 
https://github.com/SAP/ui5-webcomponents-react/tree/main/examples/vite-ts

We just haven't mentioned it in our docs yet as we wanted to test it with some pilot projects first, and so far it's looking good 🙂 
Please feel free to test it out - any feedback is highly appreciated!
